src/types.js

```diff
@@ -0,0 +1,9 @@
+import PropTypes from 'prop-types';
+
+export const TypeMovie = PropTypes.shape({
+  title: PropTypes.string.isRequired,
+  description: PropTypes.string.isRequired,
```

Comment:
According to the task, `description` wasn't required so you could set just a default value for it